src/awscli_login/saml.py

```diff
@@ -146,7 +152,7 @@ def refresh(url: str, cookies: str) -> Tuple[str, List[Role]]:
     except FileNotFoundError:
         raise MissingCookieJar(url)
 
-    soap = saml_login(url, jar)
+    soap = saml_login(url, jar, verify_cert)
```

Comment:
I don't understand why MyPy is asking for Optional[str] here, seems like `bool` with a default to `True` should do.

```suggestion
soap = saml_login(url, jar,  verify_cert: bool = True)
```